src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -46,6 +51,12 @@ class FileMetadataService(fileMetadataRepository: FileMetadataRepository,
             case None => throw new IllegalStateException(s"Cannot find client side checksum for file ${addFileMetadataInput.fileId}")
           }
           fileStatusRow: FilestatusRow = FilestatusRow(uuidSource.uuid, addFileMetadataInput.fileId, Checksum, fileStatus, timestamp)
+          _ <- Future {
+            logger.info("File check {} for fileId {} completed with status {}",
```

Comment:
If the structure of these three messages needs to be kept consistent for querying, then could you extract a shared method to do it?

Reply:
Yeah, makes sense. I've created a LoggingUtils class and moved it in there.